minecraft_servers/suchtcraft/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "server_name": "suchtcraft",
+  "nice_name": "SuchtCraft",
+  "direct_ip": "SuchtCraft.net",
+  "social": {
+    "web": "https://suchtcraft.net/",
+    "web_support": "https://suchtcraft.net/forum/",
```

Comment:
Remove "/" at the end.

Reply:
Changed with 1f74679de7ddc16a9434201b6b636a6c464ce71a